examples/gno.land/p/demo/uuid/uuid.gno

```diff
@@ -0,0 +1,132 @@
+// Package uuid provides utility functions for generating unique identifiers
+// (UUIDs) based on a custom implementation inspired by the Snowflake algorithm.
+// The UUIDs generated by this package ensure uniqueness and are suitable for
+// distributed systems. The package offers a mechanism to convert these IDs
+// to standard UUID format strings, making them compatible with systems
+// requiring UUIDs.
+// add import "gno.land/p/demo/entropy" to use the entropy package. For of the pseudo-random number generator.
+
+package uuid
+
+import (
+	"encoding/binary"
+	"std"
+	"strings"
+	"time"
+
+	"gno.land/p/demo/entropy"
+	"gno.land/p/demo/ufmt"
+)
+
+const (
+	TimestampLength uint8  = 41
```

Comment:
Where did you get these numbers?

Reply:
These numbers come from [Snowflake](https://pkg.go.dev/github.com/godruoyi/go-snowflake#section-sourcefiles)